bittensor_cli/cli.py

```diff
@@ -3532,14 +3532,14 @@ def subnets_pow_register(
         # TODO add the following to config
         processors: Optional[int] = typer.Option(
             defaults.pow_register.num_processes,
-            "-processors",
-            "-p",
+            "--processors",
+            "--p",
```

Comment:
No man. If the `-p` conflicts with `--wallet-path` alias, just remove this `--p` alias entirely. There's no reason for a single char option to be a `--`.